MijnApp-Frontend-React/src/components/CreateAccount.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Container, Button, Grid, Link } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import ArrowRightAltIcon from '@material-ui/icons/ArrowRightAlt';
+
+function ArrangeYourself() {
+  const useStyles = makeStyles((theme) => ({
+    root: {
+      paddingTop: 0,
+    },
+    menuButton: {
+      marginRight: theme.spacing(2),
+    },
+    title: {
+      flexGrow: 1,
+    },
+    container: {
+      backgroundColor: "#f9f9f9",
+      height: "90vh",
+      justifyContent:"cente"
+    },
+    wrapIcon: {
+      verticalAlign: "middle",
+    },
+    buttonOrange: {
+      backgroundColor: theme.palette.warning.dark + "!important",
+      height: 40,
+      width: "95%",
+      textTransform: "none",
+      marginTop: "2px",
+      color: "white",
+      fontWeight: "bold",
+      justifyContent: "center"
+    },
+    image: {
+      marginLeft: "10px",
+      marginTop: "3px",
+      height: 20,
+    },
+    screenImage:{
+        height: 350,
+        width: 300,
+        alignItems: "center"
+    }
+  }));
+
+  const classes = useStyles();
+  return (
+    <Container maxWidth="sm" className={classes.container}>
+      <Grid
+        container
+        direction="column"
+      >
+        <Grid container alignItems="flex-start" >
+          <Grid container >Welkomstour
+          <Link href="login" float="Right">
+              Overlaan
+            </Link>
+            </Grid>
```

Comment:
For automatic correction paddings and code style you can set up prettier
https://prettier.io/